### PR TITLE
edited printf option

### LINKED ISSUE #55: printf statements capitalized not consistent
in some printf's capitalized letters not included!

### PR DESCRIPTION
**Pull request by ishansivmal**
## 📝 Description

edited printf in ecah funtion

**Fixes #55**

## 🚀 Type of change

Please delete options that are not relevant.

- [x] ✨ New feature (non-breaking change which adds functionality)

## ✅ Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## 🖼️ Screenshots:

- [ ] I have added screenshots that prove my fix is effective or that my feature works

### CODE REVIEW

File: func.c [hunk 1]
```diff
@@ -33,9 +33,9 @@ int login()
     char password[100];
 
     printf("\n");
-    printf("=======================================\n");
+    printf("========================================\n");
```
Comment (ktauchathuranga) [inline func.c:36]: its extra = sign, check again :(

File: func.c [hunk 2]
```diff
@@ -49,12 +49,12 @@ int login()
...
         strcmp(stored_password_hash, entered_password_hash) == 0)
     {
-        printf("[+] Login successful!\n");
+        printf("[+] Login Successful!\n");
         return 1;
     }
     else
     {
-        printf("[!] Login failed. Please check your username and password... :(\n");
+        printf("[!] Login failed. Please check your username and password correctly... :(\n");
```
Comment (ktauchathuranga) [inline func.c:57]: make capital, ALL letters (first letter of the word)

File: func.c [hunk 3]
```diff
@@ -330,7 +330,7 @@ void displayMenu()
     // these are just place holders, TO BE CHANGED!
     printf("\n");
     printf("======================================\n");
-    printf("=       Tuition Class Management     =\n");
+    printf("=|       Tuition Class Management     |=\n");
```
Comment (ktauchathuranga) [inline func.c:333]: check again the alignment

File: func.c [hunk 4]
```diff
@@ -575,7 +575,7 @@ void status()
 
     printf("\n");
     printf("=======================================\n");
-    printf("=               Summary               =\n");
+    printf("=               SUMMARY              =\n");
```
Comment (ktauchathuranga) [inline func.c:578]: first letter only

**Review by ktauchathuranga — CHANGES_REQUESTED** [review]
check the added reviews and fix them.
:)

File: func.c (hunk 2 above)
Comment (ishansivmal) [reply func.c]: done... :)

File: func.c (hunk 3 above)
Comment (ishansivmal) [reply func.c]: DONEEE

File: func.c (hunk 1 above)
Comment (ishansivmal) [reply func.c:36]: DONE HALO

File: func.c (hunk 4 above)
Comment (ishansivmal) [reply func.c]: haduwa anei...

**Review by ktauchathuranga — APPROVED** [review]
good job....
:)

**Pull request merged**